### ISSUE TITLE
how to send torrents

### ISSUE THREAD

**Issue by ccorcos** [maxogden/torrent#22]
I'm not too familiar with torrents, but I see that you can create a torrent with this program... How can I send this file to someone else? is that possible through this program?

**Comment by maboiteaspam** [comment]
hi,

give a look to https://github.com/feross/bittorrent-protocol

**Comment by No9** [comment]
Hi @ccorcos 
Late to the party but in case anyone else is looking I put together a workshopper on using this tool 
https://github.com/No9/torrential 

I hope some folks find it useful

**Comment by maxogden** [comment]
@no9 you should put that on nodeschool.io :)

**Comment by No9** [comment]
@maxogden Yes I added this issue a while ago https://github.com/nodeschool/workshoppers/issues/34 
But it doesn't seem to be getting anywhere :(
